### PR TITLE
Adding a file about Los Angeles,CA

### LINKED ISSUE #61: Create a new file for alexladines hometown
## Add file for Los Angeles, CA
- [x] Create a branch
- [x] Add my file
- [x] Create a commit
- [x] Open a pull request
- [x] Have a discussion about the pull request 
- [x] Merge our pull request

### PR DESCRIPTION
**Pull request by alexladines**
Which happens to be @alexladines hometown 

This pull request closes #61

### CODE REVIEW

File: places/Los-Angeles.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Los Angeles, CA
+
+### Great Places to Eat
```
Comment (alexladines) [inline places/Los-Angeles.md:3]: Add some bomb food places! 👍

Comment (SoidaPalacios) [inline places/Los-Angeles.md:3]: > Do you like the Ivy?

@hans-zen I've actually never been there but I just googled it and it looks good!

File: places/Los-Angeles.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+### Fun Things to Do
```
Comment (thehackermonkey) [inline places/Los-Angeles.md:7]: > traveling there on summer. 👍 

Where in LA specifically? 💯

**Comment by thehackermonkey** [comment]
:shipit: go go go!

**Pull request merged**